packages/client/src/views/SysAdmin/Performance/Vsexports/VSExport.tsx

```diff
@@ -0,0 +1,180 @@
+ *
+ * OpenCRVS is also distributed under the terms of the Civil Registration
+ * & Healthcare Disclaimer located at http://opencrvs.org/license.
+ *
+ * Copyright (C) The OpenCRVS Authors. OpenCRVS and the OpenCRVS
+ * graphic logo are (registered/a) trademark(s) of Plan International.
+ */
+import React from 'react'
+import { SysAdminContentWrapper } from '@client/views/SysAdmin/SysAdminContentWrapper'
+import { FormTabs } from '@opencrvs/components/lib/forms'
+import styled from 'styled-components'
+import { useIntl } from 'react-intl'
+import { BodyContent } from '@opencrvs/components/lib/layout'
+import { Content } from '@opencrvs/components/lib/interface/Content'
+import { messages } from '@client/i18n/messages/views/config'
+import {
+  ListViewItemSimplified,
+  ListViewSimplified
+} from '@opencrvs/components/lib/interface/ListViewSimplified/ListViewSimplified'
+import { Query } from '@client/components/Query'
+import { GET_TOTAL_VSEXPORT } from './queries'
+import {
+  Label,
+  Value
+} from '@client/views/SysAdmin/Config/Application/Components'
+import { LoadingIndicator } from '@client/views/OfficeHome/LoadingIndicator'
+
+import { MINIO_URL } from '@client/utils/constants'
+import { DynamicHeightLinkButton } from '@client/views/Settings/items/components'
+import { ErrorToastNotification } from '@opencrvs/components/lib/interface'
+
+import { buttonMessages, errorMessages } from '@client/i18n/messages'
+import { Event, GetVsExportsQuery, VsExport } from '@client/utils/gateway'
+
+const UserTable = styled(BodyContent)`
+  padding: 0px;
+  margin: 8px auto 0;
+  @media (max-width: ${({ theme }) => theme.grid.breakpoints.md}px) {
+    padding: 0px;
+  }
+`
+type VSExportProps = {
+  items: VsExport[]
+}
+
+const notificationActionButtonHandler = () => {
+  window.location.reload()
+}
+
+function downloadURI(uri: string, name: string) {
+  const link = document.createElement('a')
+  link.href = uri
+  link.download = name
+  document.body.appendChild(link)
+  link.click()
+  document.body.removeChild(link)
+}
+
+function TabContent(props: VSExportProps) {
+  const intl = useIntl()
+  const items: VsExport[] = props.items
+  return (
+    <>
+      {items.map((item: VsExport) => {
+        const fileName = intl.formatMessage(messages.vitalStatisticsExport, {
+          year: item.year,
+          event: item.event,
+          fileSize: ''
+        })
+        const sizeValue = intl.formatMessage(messages.vitalStatisticsExport, {
+          year: item.year,
+          event: item.event,
+          fileSize: item.fileSize
+        })
+
+        return (
+          <ListViewSimplified key={`${item.createdOn}_${item.event}`}>
+            <ListViewItemSimplified
+              label={<Label id={`${item.year}_label`}>{item.year}</Label>}
+              value={<Value id={`${item.createdOn}_value`}>{sizeValue}</Value>}
+              actions={
+                <DynamicHeightLinkButton
```

Comment:
where is this defined?
Can we not just use < Link /> from the component library?